Error compiling

I tried to compile following the README, however I got the following error:
```
-- The C compiler identification is GNU 7.3.0
-- The CXX compiler identification is unknown
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
CMake Error at CMakeLists.txt:2 (project):
  No CMAKE_CXX_COMPILER could be found.
```

Installing **g++** using this ``` sudo apt-get install g++``` resolved the issue.
May be adding it to the README would be good ?

--- a/README.md
+++ b/README.md
@@ -4,6 +4,7 @@ hanabi\_learning\_environment is a research platform for Hanabi experiments. The
 
 ### Getting started
 ```
+sudo apt-get install g++         # if you don't already have a CXX compiler
 sudo apt-get install cmake       # if you don't already have CMake
 sudo apt-get install python-pip  # if you don't already have pip
 pip install cffi                 # if you don't already have cffi

